scripts/barclamp_mgmt_lib.rb

```diff
@@ -408,89 +390,47 @@ def bc_install_layout_1_app(from_rpm, bc, bc_path, yaml)
 
     if dirs.include? 'crowbar_framework'
       debug "path entries include \"crowbar_framework\""
-      files += bc_cloner('crowbar_framework', bc, nil, bc_path, BASE_PATH, false)
-      framework_permissions bc, bc_path
+      files += bc_cloner('crowbar_framework', nil, bc_path, BASE_PATH)
+      framework_permissions
     end
 
     if dirs.include? 'bin'
       debug "path entries include \"bin\""
-      files += bc_cloner('bin', bc, nil, bc_path, BASE_PATH, false)
+      files += bc_cloner('bin', nil, bc_path, BASE_PATH)
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)